src/mlj_embedder_interface.jl

```diff
@@ -129,25 +133,56 @@ X = (;
         repeat(["group1", "group1", "group2", "group2", "group3"], Int(N / 5)),
     ),
 )
-y = categorical([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])           # Classification
+y = categorical(repeat(["class1", "class2", "class3", "class4", "class5"], Int(N / 5)))
 
-# Initiate model
-EntityEmbedder = @load EntityEmbedder pkg=MLJFlux
+# Load the entity embedder, it's neural network backbone and the SVC which inherently supports
+# only continuous features
+EntityEmbedder = @load EntityEmbedder pkg=MLJFlux   
 NeuralNetworkClassifier = @load NeuralNetworkClassifier pkg=MLJFlux
+SVC = @load SVC pkg=LIBSVM              
 
-clf = NeuralNetworkClassifier(embedding_dims=Dict(:Column2 => 2, :Column3 => 2))
 
-emb = EntityEmbedder(clf)
+
+emb = EntityEmbedder(NeuralNetworkClassifier(embedding_dims=Dict(:Column2 => 2, :Column3 => 2)))
+clf = SVC(cost = 1.0)
+
+julia> pipeline = emb |> clf
+DeterministicPipeline(
```

Comment:
Is there a reason we don't get Documenter to genera the output here? Ie., should we not make this a `@example` fenced block rather than plain `julia` fenced block?

Reply:
Or just omit output altogether.